core/server/src/main/java/alluxio/worker/file/UnderFileSystemManager.java

```diff
@@ -57,13 +61,17 @@
      * Constructor for an input stream agent for a UFS file. The file must exist when this is
      * called.
      *
+     * @param sessionId the session this agent belongs to
+     * @param id the id of the agent
```

Comment:
consider calling this agentId now that we have sessionId. I don't feel strongly about this